10/action.yml

```
@@ -0,0 +1,19 @@
+name: Building JS
+author: Poulinho
```

```suggestion
author: PrestaShop
```

Your contributions will be kept as commits anyway.